components/Comments.tsx

```diff
@@ -0,0 +1,41 @@
+import { Divider, Spacer, Text } from '@geist-ui/react';
+import { formatRelative, parseISO } from 'date-fns';
+
+import { Comment } from '../redux/types';
+import Person from './Person';
+
+interface CommentsParams {
+  comments?: Comment[];
+}
+
+const Comments = ({ comments = [] }: CommentsParams): JSX.Element => {
+  if (!comments.length) {
+    return (
+      <>
+        <Divider>No Comments</Divider>
+        {/* Todo: Add Comment Box */}
+      </>
+    );
+  }
+  return (
+    <>
+      <Divider>Comments</Divider>
+      {comments.map(
+        (comment: Comment): JSX.Element => (
```

Comment:
Here typescript will know it's a comment type, I know there's dev's that loves to put more explicit types into it, but I'm more in favor of, if typescript knows then fewer codes is enough. 😬

```suggestion
        (comment): JSX.Element => (
```

Reply:
I don't mind being extra terse in some situations